examples/quickstart/nextjs/app/api/[...nile]/nile.ts

```diff
@@ -1,8 +1,9 @@
 import { Nile } from "@niledatabase/server";
+import { nextJs } from "@niledatabase/nextjs";
 
-export const nile = await Nile({
-  secureCookies: process.env.VERCEL === "1",
+const nile = await Nile({
   debug: true,
+  extensions: [nextJs],
```

Comment:
with the advent of the nextjs middleware, we no longer need `configureInstance`, thank goodness. Also too the opportunity to upgrade to next 15.3 (with the async params), and react 19.1, which I think was the last thing on 18, I think.